tests/parsers/text_plugins/bash_history.py

```diff
@@ -44,6 +44,17 @@ def _TestEventsFromFile(
     event_data = storage_writer.GetAttributeContainerByIndex('event_data', 0)
     self.CheckEventData(event_data, expected_event_values)
 
+    # Test multiline
+    expected_event_values = {
```

Comment:
style nit: put `(` after command or indent string on the same level as the start of the string.